### PR TITLE
feat(pdf): Upgrade pdf.js to v2.6.347 and closure compiler to v20200719

### PR DESCRIPTION
**Pull request by jstoffan**
Todo
- [x] Additional document testing
- [x] Cross-browser testing

### CODE REVIEW

File: build/karma.conf.js [hunk 1]
```diff
@@ -92,7 +83,6 @@ module.exports = config =>
         files: [
             'https://cdn01.boxcdn.net/polyfills/core-js/2.5.3/core.min.js',
             `src/third-party/model3d/${MODEL3D_STATIC_ASSETS_VERSION}/three.min.js`,
-            `src/third-party/doc/${DOC_STATIC_ASSETS_VERSION}/**/*.js`,
```
Comment (ConradJChan) [inline build/karma.conf.js:95]: Intentional?

File: src/lib/viewers/doc/DocBaseViewer.js [hunk 2]
```diff
@@ -747,6 +747,7 @@ class DocBaseViewer extends BaseViewer {
         const { file, location } = this.options;
         const assetUrlCreator = createAssetUrlCreator(location);
         const hasDownload = checkPermission(file, PERMISSION_DOWNLOAD);
+        const hasForms = hasDownload && this.getViewerOption('disableForms') === false;
```
Comment (ConradJChan) [inline src/lib/viewers/doc/DocBaseViewer.js:750]: Does this need a test? Also does this warrant an update to the preview sdk readme?

File: build/karma.conf.js (hunk 1 above)
Comment (jstoffan) [reply build/karma.conf.js:95]: Yep, intentional! PhantomJS chokes on the latest pdf.js assets and they aren't actually necessary, anyway. We only have one test that depended on them, so I just moved it over to a simple mock object.

File: src/lib/viewers/doc/DocBaseViewer.js (hunk 2 above)
Comment (jstoffan) [reply src/lib/viewers/doc/DocBaseViewer.js:750]: Hm, I think I'll just set it to false.

**Review by ConradJChan — APPROVED** [review]

**Review by ConradJChan — APPROVED** [review]

**Review by ConradJChan — APPROVED** [review]

**Pull request merged**